examples/cpp/BasicOperations/startserver.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -f
+GFSH_PATH=""
+which gfsh 2> /dev/null
+
+if [ $? -gt 0 ]; then
+    GFSH_PATH=`which gfsh 2> /dev/null`
+fi
+
+
+if [ "$GFSH_PATH" == "" ]; then
+    if [ "$GEODE_HOME" == "" ]; then
+        echo "Could not find gfsh. Please set the GEODE_HOME path."
+        echo "e.g. export GEODE_HOME=<path to Geode>"
+    else
+        GFSH_PATH=$GEODE_HOME/bin/gfsh
+    fi
+fi
+
+$GFSH_PATH  -e "start locator --name=locator" -e "start server --name=server"  -e "create region --name=exampleRegion --type=PARTITION"
+
+
+
+
+
+
+
+
+
```

Comment:
Unless we're getting paid for line-count, let's trim extra lines.  It's an eyesore.

Reply:
Magic number should be `#!/bin/env bash`.

Why get the path to gfsh if its in the path? If `which` finds it then its in the path so you can just exec it. If it isn't in the path then you need the `GEODE_HOME` set to assume it is in `${GEODE_HOME}/bin`.

Reply:
Since the user has to know the path to `gfsh` I still fail to see the value in creating this script.

start.gfsh
```
#!/bin/env gfsh

start locator --name=locator
start server --name=server
create region --name=exampleRegion --type=PARTITION
```
./start.gfsh

or run `gfsh` can copy and paste the contents of this script.

Reply:
The region it creates should match the name of the project.

Reply:
File is missing license header.

Reply:
done

Reply:
Done. The story says we should deal with orders. So I called it orders.